### DISCUSSION TITLE
Rails 7 and React 18 Setup or installation guide

### DISCUSSION THREAD

**Discussion by lakshmaji** [inertiajs/inertia#1328] [Show and tell]
I have created a starter template for using `inertia.js` with Rails 7, [here][lakshmaji-inertia-repo] with minimal external dependencies, below are list of tools or frameworks I have used,

1. React 18 
2. Rails 7
3. Vite (esbuild)

I have added a step by step guide for setting up `Inertia` with `Rails 7` [here][inertia-blog].

I have made the [repo][lakshmaji-inertia-repo] as `template`, which means the template can be used to start new projects from it. Or you can follow the commit history.

So far this is basically a client side application (views only). I will add another guide for SSR walk through stay tuned. 

Thank you Inertia 🙏.

[inertia-blog]: https://lakshmaji.medium.com/rails-7-reactjs-setup-monolith-b9baa0f84271
[lakshmaji-inertia-repo]: https://github.com/lakshmaji/rails-react

**Comment by marcelkooi** [comment]
Hi @lakshmaji! It looks like the links you posted are dead. Is this the repo? https://github.com/lakshmaji/rails-react

**Reply by lakshmaji** [reply]
My bad, There is a typo in linking. replaced `(` with `[`.

Thank you @marcelkooi
